samza-core/src/main/java/org/apache/samza/clustermanager/container/placements/ContainerPlacementStatus.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.samza.clustermanager.container.placements;
+
+
+/**
+ * Encapsulates state related to progress of a control action which should be passed between the
+ * {@link org.apache.samza.clustermanager.ContainerManager} and any external controllers for reporting state of an action
+ */
+public class ContainerPlacementStatus {
+  public enum StatusCode {
+    CREATED, BAD_REQUEST, ACCEPTED, IN_PROGRESS, SUCCEEDED, FAILED
```

Comment:
Can you add meaning for each of these states?
Created = request has been recorded but no action has been taken.
Bad-request = request has been recorded, processed, and deemed a bad due to ......
Accepted = request has been recorded, and a resource request for it has been made??
In-progress = resource request has been met ??
succeeded = complete?
failed = ?????
Do we need all of these?